
From: Paul Jackson <pj@sgi.com>

Builds of 2.6.9-rc1-mm5 ia64 NUMA configs fail, with many complaints that
SD_NODE_INIT is defined twice, in asm/processor.h and linux/sched.h.

I guess that the preprocessor conditionals were wrong when Nick added the
per-arch override ability again of SD_NODE_INIT were wrong.  At least this
change lets me rebuild ia64 again.

Signed-off-by: Paul Jackson <pj@sgi.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/sched.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN include/linux/sched.h~sched-make-domain-setup-overridable-fix include/linux/sched.h
--- 25/include/linux/sched.h~sched-make-domain-setup-overridable-fix	2004-09-26 17:18:25.729746936 -0700
+++ 25-akpm/include/linux/sched.h	2004-09-26 17:18:25.741745112 -0700
@@ -531,7 +531,7 @@ extern void cpu_attach_domain(struct sch
 	.nr_balance_failed	= 0,			\
 }
 
-#ifdef CONFIG_NUMA
+#if defined(CONFIG_NUMA) && !defined(SD_NODE_INIT)
 #define SD_NODE_INIT (struct sched_domain) {		\
 	.span			= CPU_MASK_NONE,	\
 	.parent			= NULL,			\
_
